compiler-core/src/javascript/tests.rs

```diff
@@ -40,7 +40,11 @@ macro_rules! assert_js_with_multiple_imports {
     ($(($name:literal, $module_src:literal)),+; $src:literal) => {
         let compiled =
             $crate::javascript::tests::compile_js($src, vec![$((CURRENT_PACKAGE, $name, $module_src)),*]).expect("compilation failed");
-        let output = format!("----- SOURCE CODE\n{}\n\n----- COMPILED JAVASCRIPT\n{}", $src, compiled);
+            let mut output = String::from("----- SOURCE CODE\n");
+            for (name, src) in [$(($name, $module_src)),*] {
```

Comment:
I don't know Rust macros very well, so I've just made this for loop. Is there a way to do this loop statically?

Reply:
Probably but I don't care